I2C.cpp

```diff
@@ -0,0 +1,24 @@
+#include "I2C.h"
+
+I2C::I2C(I2C_HandleTypeDef* handler, uint16_t endereco, uint32_t delay){
+    this->handler = handler;
+    this->endereco = endereco;
+    this->delay = delay;
+}
+
+
+I2C::~I2C() {
+    HAL_I2C_DeInit(this->handler);
+}
+
+
+uint8_t *I2C::ler(uint16_t tam_registrador, uint16_t registrador, uint16_t tamanho) {
+    uint8_t *dados = (uint8_t*)malloc(tamanho * sizeof(uint8_t));
+	HAL_I2C_Mem_Read(this->handler, this->endereco, registrador, tam_registrador, dados, tamanho, this->delay);
+	return dados;
```

Comment:
Faz a boa troca o tab chernobola da IDE por 4 espaço <3

```suggestion
    HAL_I2C_Mem_Read(this->handler, this->endereco, registrador, tam_registrador, dados, tamanho, this->delay);
    return dados;
```